src/Symfony/Component/Process/ExecutableFinder.php

```diff
@@ -46,26 +46,17 @@ public function addSuffix(string $suffix)
      */
     public function find(string $name, string $default = null, array $extraDirs = []): ?string
     {
+        $dirs = [
+            explode(\PATH_SEPARATOR, getenv('PATH') ?: getenv('Path')),
+            $extraDirs
+        ];
+
         if (\ini_get('open_basedir')) {
-            $searchPath = array_merge(explode(\PATH_SEPARATOR, \ini_get('open_basedir')), $extraDirs);
-            $dirs = [];
-            foreach ($searchPath as $path) {
-                // Silencing against https://bugs.php.net/69240
-                if (@is_dir($path)) {
-                    $dirs[] = $path;
-                } else {
-                    if (basename($path) == $name && @is_executable($path)) {
-                        return $path;
-                    }
-                }
-            }
-        } else {
-            $dirs = array_merge(
-                explode(\PATH_SEPARATOR, getenv('PATH') ?: getenv('Path')),
-                $extraDirs
-            );
+            $dirs[] = explode(\PATH_SEPARATOR, \ini_get('open_basedir'));
```

Comment:
the open_basedir should not be added as a search path. It is not a place to look for executable. that's not what `open_basedir` is about

Reply:
This is only for backwards compatibility. I am perfectly fine with removing it as I agree that `open_basedir` is not about finding executables (as described in the PR ddescription) - nevertheless this is how it is currently implemented. So please confirm that I should remove it and I will do...

Reply:
The old code has been reported as buggy multiple times due to misusing `open_basedir`

Reply:
Thanks, removed it.